genie-core/src/test/java/com/netflix/genie/core/jobs/JobKillReasonFileTest.java

```diff
@@ -0,0 +1,37 @@
+package com.netflix.genie.core.jobs;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.genie.test.categories.UnitTest;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.io.IOException;
+
+/**
+ * Unit tests for the JobKillReasonFile class.
+ *
+ * @author mprimi
+ * @since 3.0.8
+ */
+@Category(UnitTest.class)
+public class JobKillReasonFileTest {
+
+    /**
+     * Test serialization and deserialization of JobKillReasonFile.
+     * @throws IOException in case of serialization error
+     */
+    @Test
+    public void serializeThenLoad() throws IOException {
+
+        final ObjectMapper objectMapper = new ObjectMapper();
+
+        final JobKillReasonFile orignalJobKillReasonFile = new JobKillReasonFile("Test");
```

Comment:
Break the actual reason out into a String and reference that variable in the assertion rather than using the getter in both places since this leads to possible test that is only succeeding due to some weird behavior in the getter. Minor and silly but still.